.github/workflows/pytest-workflow.yml

```diff
@@ -29,7 +29,7 @@ env:
   NXF_SINGULARITY_LIBRARYDIR: ${{ github.workspace }}/.singularity
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   # renovate: datasource=github-releases depName=nextflow-io/nextflow versioning=semver
```

Comment:
I really need to clean up the renovate stuff so we can get these auto bumped 🫠

Reply:
And we need to finish migrating pytests tests